src/runtime/scheduler_tinygoriscv.S

```diff
@@ -0,0 +1,29 @@
+.section .text.tinygo_scanCurrentStack
+.global  tinygo_scanCurrentStack
+.type    tinygo_scanCurrentStack, %function
+tinygo_scanCurrentStack:
+   // Push callee-saved registers onto the stack.
+   addi sp, sp, -64
+   sw ra,  60(sp)
+   sw s11, 56(sp)
+   sw s10, 52(sp)
+   sw s9,  48(sp)
+   sw s8,  44(sp)
+   sw s7,  40(sp)
+   sw s6,  36(sp)
+   sw s5,  32(sp)
+   sw s4,  28(sp)
+   sw s3,  24(sp)
+   sw s2,  20(sp)
+   sw s1,  16(sp)
+   sw s0,  12(sp)
+
+   // Scan the stack.
+   addi a0, sp, 0
```

Comment:
You could perhaps use `mv a0, sp` here for better readability (it's an alias).

Reply:
I made the change, and confirmed that it builds. Do you have a board which you can test this on?

Reply:
I will test this shortly.

Reply:
Confirmed that the changes to this PR work as expected on my risc-v board.

Reply:
For the future, you can run the test in an emulator, for example `tinygo run -target=riscv-qemu ./testdata/gc.go`. There is another (`-target=hifive1-qemu`) but I accidentally broke it (fixed in https://github.com/tinygo-org/tinygo/pull/1026).